"use strict";(self.webpackChunkpadel_app=self.webpackChunkpadel_app||[]).push([[4719],{94719:r=>{r.exports=JSON.parse('{"default":{"success":"Proceso finalizado con \xe9xito","error":"Algo sali\xf3 mal.","empty":"A\xfan no hay datos para mostrar."},"login":{"401":"Error de inicio de sesi\xf3n. El nombre de usuario o la contrase\xf1a que proporcionaste no coinciden con nuestros registros.","error":"Error de inicio de sesi\xf3n debido a un error interno, por favor int\xe9ntalo de nuevo. Si el error persiste, contacta a nuestro equipo de soporte."},"activityLogTable":{"empty":"A\xfan no hay contenido para mostrar","noMatch":"No hay resultados para tu b\xfasqueda","error":{"title":"Error al cargar","description":"El registro de actividad no pudo ser cargado debido a un error interno. Por favor, int\xe9ntalo de nuevo m\xe1s tarde."}},"customersTable":{"empty":"A\xfan no hay clientes para mostrar.","noMatch":"No hay resultados para tu b\xfasqueda","error":{"title":"Error al cargar","description":"La lista de clientes no pudo ser cargada debido a un error interno. Por favor, int\xe9ntalo de nuevo m\xe1s tarde."}},"alerts":{"empty":"No hay alertas","error":{"title":"Error al cargar","description":"La lista de alertas no pudo ser cargada debido a un error interno. Por favor, int\xe9ntalo de nuevo m\xe1s tarde."}},"createUser":{"successTitle":"Usuario creado con \xe9xito","success":"Por favor, env\xeda las credenciales al nuevo usuario.","errorTitle":"Error interno","error":"El operador no pudo ser creado, por favor int\xe9ntalo de nuevo. Si el error persiste, contacta a nuestro equipo de soporte."},"editUser":{"successTitle":"El usuario {{userId}} ha sido editado con \xe9xito","success":"Por favor, env\xeda las nuevas credenciales al usuario.","errorTitle":"Error interno","error":"El usuario no pudo ser editado, por favor int\xe9ntalo de nuevo. Si el error persiste, contacta a nuestro equipo de soporte."},"deleteUser":{"success":"El usuario {{userId}} ha sido eliminado con \xe9xito","errorTitle":"Error interno","error":"El usuario no pudo ser eliminado, por favor int\xe9ntalo de nuevo. Si el error persiste, contacta a nuestro equipo de soporte."},"dismissAll":{"error":"Las alertas no pudieron ser descartadas, por favor int\xe9ntalo de nuevo. Si el error persiste, contacta a nuestro equipo de soporte."},"dismiss":{"error":"La alerta no pudo ser descartada, por favor int\xe9ntalo de nuevo. Si el error persiste, contacta a nuestro equipo de soporte."},"operatorNewCustomer":{"success":"El cliente {{customer}} ha sido creado con \xe9xito.","errorTitle":"Error al cargar","error":"El cliente {{customer}} no pudo ser creado, por favor int\xe9ntalo de nuevo. Si el error persiste, contacta a nuestro equipo de soporte."}}')}}]);